types/react-dom/client.d.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * WARNING: This entrypoint is only available starting with `react-dom@18.0.0-rc.1`
+ */
+
+// See https://github.com/facebook/react/blob/main/packages/react-dom/client.js to see how the exports are declared,
+
+import React = require('react');
+export interface HydrationOptions {
+    onHydrated?(suspenseInstance: Comment): void;
+    onDeleted?(suspenseInstance: Comment): void;
```

Comment:
These two APIs aren't actually exposed so they should be removed.

Reply:
Yeah these were always odd to me but I thought I checked that these were reachable by the public entrypoint.